zeroex/orderwatch/order_watcher_test.go

```diff
@@ -32,6 +34,8 @@ const (
 	blockWatcherRetentionLimit  = 20
 	blockPollingInterval        = 1000 * time.Millisecond
 	ethereumRPCMaxContentLength = 524288
+	maxEthRPCRequestsPer24HrUTC = 1000000
+	maxEthRPCRequestsPerSeconds = float64(1000.0)
```

Comment:
Again I don't think you need the `float64` cast here but maybe I'm missing something.